src/app/(proper_react)/layout.tsx

```diff
@@ -3,30 +3,18 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import { ReactNode } from "react";
-import { getServerSession } from "next-auth";
-import { getL10n, getL10nBundles, getLocale } from "../functions/server/l10n";
+import { getL10nBundles, getLocale } from "../functions/server/l10n";
 import { L10nProvider } from "../../contextProviders/localization";
-import { authOptions } from "../api/utils/auth";
-import { Shell } from "./Shell";
-import {
-  ReactAriaI18nProvider,
-  ReactAriaSsrProvider,
-} from "../../contextProviders/react-aria";
+import { ReactAriaI18nProvider } from "../../contextProviders/react-aria";
 
 export default async function Layout({ children }: { children: ReactNode }) {
   const l10nBundles = getL10nBundles();
-  const l10n = getL10n(l10nBundles);
-  const session = await getServerSession(authOptions);
 
   return (
     <L10nProvider bundleSources={l10nBundles}>
-      <ReactAriaSsrProvider>
```

Comment:
`ReactAriaSsrProvider` was deprecated in the [latest release of React Aria](https://react-spectrum.adobe.com/releases/2023-06-27.html). The `<Shell>` was moved into the `/redesign/` directory to ensure it wouldn't wrap the onboarding.